From cfb1b589b8931f48787ac67675f179e89e76cd95 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal.vasilek@nic.cz>
Date: Tue, 18 Apr 2023 13:19:42 +0200
Subject: [PATCH] bind: disable geoip because of maxminddb dep

the libmaxminddb package recently added a pkgconfig file which caused
the geoip feature to be enabled automatically and the package to fail
when it has a new unexpected dependency

Signed-off-by: Michal Vasilek <michal.vasilek@nic.cz>
---
 net/bind/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/bind/Makefile b/net/bind/Makefile
index 986e64e9ef..1b79bba824 100644
--- a/net/bind/Makefile
+++ b/net/bind/Makefile
@@ -136,6 +136,7 @@ TARGET_LDFLAGS += -Wl,--gc-sections,--as-needed
 
 CONFIGURE_ARGS += \
 	--disable-linux-caps \
+	--disable-geoip \
 	--with-openssl="$(STAGING_DIR)/usr" \
 	--with-libtool \
 	--without-lmdb \
-- 
2.40.0

